dali/operators/image/convolution/gaussian_blur_params.h

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_OPERATORS_IMAGE_CONVOLUTION_GAUSSIAN_BLUR_PARAMS_H_
+#define DALI_OPERATORS_IMAGE_CONVOLUTION_GAUSSIAN_BLUR_PARAMS_H_
+
+#include <vector>
+
+#include "dali/pipeline/data/views.h"
+#include "dali/pipeline/operator/common.h"
+
+namespace dali {
+
+int GaussianSigmaToDiameter(float sigma) {
+  return 2 * ceilf(sigma * 3) + 1;
+}
+
+float GaussianDiameterToSigma(int diameter) {
+  // Based on OpenCV
+  int radius = (diameter - 1) / 2;
+  return (radius - 1) * 0.3 + 0.8;
+}
+
+struct GaussianDimDesc {
```

Comment:
This name is strange - if you insist on making it specific, I'd suggest GaussianBlurDimDesc - because this desc is not Gaussian.

Reply:
Oh, that one sneaked from the Op